network/linuxbridge_endpointclient_linux.go

```diff
@@ -0,0 +1,98 @@
+package network
+
+import (
+	"net"
+
+	"github.com/Azure/azure-container-networking/ebtables"
+	"github.com/Azure/azure-container-networking/log"
+	"github.com/Azure/azure-container-networking/netlink"
+)
+
+type LinuxBridgeEndpointClient struct {
```

Comment:
Lets call it just bridge (rename file as well -> bridge_endpointclient_linux.go)).